### ISSUE TITLE
Jupyter lab

### ISSUE THREAD

**Issue by jni** [scipy-2023-image-analysis/tutorial#13]
Hmmm, looks like the jupytext experience is not actually as seamless for jupyter lab users:

https://jupytext.readthedocs.io/en/latest/text-notebooks.html#text-notebooks

We need to add a link in the instructions to that page... Unless either of you knows an easier way @lagru @kne42? It feels like this should be easier in 2023... 😅

**Comment by jni** [comment]
ok it's a right-click away on *most* browsers... My firefox blocks right-click capture so I couldn't see it... 😅 And I've forgotten where to change that setting... 😂

**Comment by jni** [comment]
Found the setting! Ignore me! 😂

**Comment by lagru** [comment]
Yes, that's why I suggested jupyter notebook to be included in the environment as well.

Slightly related, some functionality for jupytext is only reachable in jupyter lab via the command palette (ctrl + shift + c). The palette seems broken for me currently, it's not reacting to input. Could you perhaps confirm that this is a me issue again and not something with our dependencies?

**Comment by jni** [comment]
I'll run it locally tomorrow but fyi it's also usable by right-clicking on the .md files > open with > jupytext.

**Comment by lagru** [comment]
Yes, that I know and luckily works for me. I am also thinking if we want to include a [jupytext.toml](https://jupytext.readthedocs.io/en/latest/config.html) to set some useful defaults so safe everyone some possible confusion.
